app-libs/stf/src/stf_sgx.rs

```diff
@@ -57,33 +73,36 @@ impl Stf {
 		ext
 	}
 
-	pub fn get_state(ext: &mut State, getter: Getter) -> Option<Vec<u8>> {
+	pub fn get_state(ext: &mut impl SgxExternalitiesTrait, getter: Getter) -> Option<Vec<u8>> {
```

Comment:
Generalize this method signature, making it consistent with some of the other methods that already used this type of signature.